readme.md

```diff
@@ -63,15 +72,15 @@ Hello, Node!
 * **Question:** Describe the difference between these two methods.
 
 * **Your Answer:**
-
+Append adds to the end of an existing file. fs.writefile() replaces content in the file.
 ---
 
 - [ ] Imagine you want to edit the middle of a file. You can use the [fs module](https://nodejs.org/api/fs.html) and the JavaScript language.
 
 * **Question:** How would you do so?
 
 * **Your Answer:**
-
+You would need to read the file, stringafy it, make the changes and then push them.
```

Comment:
I think you mean `.parse()` here, make changes, then `.stringify()`, then write back.